tron/mcp.py

```diff
@@ -53,30 +56,34 @@ def reconfigure(self, namespace=None):
             log.exception(f"reconfigure failure: {e.__class__.__name__}: {e}")
             raise e
 
-    def _load_config(self, reconfigure=False, namespace_to_reconfigure=None):
+    def _load_config(self, reconfigure=False, namespace_to_reconfigure=None, tron_start_time=None):
```

Comment:
this will get called pretty frequently post-boot, so i don't think we want to pass through the boot time here